nx.json

```diff
@@ -0,0 +1,32 @@
+{
+  "$schema": "./node_modules/nx/schemas/nx-schema.json",
+  "defaultBase": "main",
+  "targetDefaults": {
+    "build": {
+      "dependsOn": ["^build"]
+    }
+  },
+  "release": {
+    "changelog": {
+      "projectChangelogs": false,
+      "workspaceChangelog": {
+        "file": false,
+        "createRelease": "github"
+      }
+    },
+    "projects": ["packages/react-native", "packages/virtualized-lists"],
+    "projectsRelationship": "independent",
```

Comment:
I think these can be dependent to align closer to upstream. Not a strong feeling though.